config/src/test/java/se/fortnox/reactivewizard/config/ConfigReaderTest.java

```diff
@@ -158,7 +156,7 @@ public void shouldThrowExceptionForInvalidYaml() {
             ConfigReader.fromFile("src/test/resources/testconfig-invalid.yml", EmptyConfig.class);
             fail("Expected exception, but none was thrown");
         } catch (RuntimeException exception) {
-            assertException(exception, MarkedYAMLException.class);
+            assertThat(exception).hasRootCauseInstanceOf(MarkedYAMLException.class);
```

Comment:
assertj allows us to get rid of try/catch, but that could be a later improvement:
`assertThatThrownBy(() -> ConfigReader.fromFile("src/test/resources/testconfig-invalid.yml", EmptyConfig.class)).isInstanceOf(RuntimeException.class).hasRootCauseInstanceOf(MarkedYAMLException.class);`

Reply:
Yeah, I tried to avoid rewriting too much tests unless required to keep down scope. Good catch though :)